app/models/equipment_model.rb

```diff
@@ -210,14 +210,14 @@ def number_overdue
   def available_count(date)
     # get the total number of objects of this kind then subtract the total
     # quantity currently reserved, checked-out, and overdue
-    total = equipment_objects.active.count
+    total = equipment_items.active.count
     reserved = Reservation.reserved_on_date(date)
                .not_returned.for_eq_model(self).count
     total - reserved - number_overdue
   end
 
   def available_object_select_options
```

Comment:
rename this method to `available_item_select_options` --> make sure you also change everywhere that it's used.